src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt

```diff
@@ -370,10 +371,8 @@ class HardwareExploration(
                         .let { tuning.tune(it, hardware, scale) }
                 })
                 .loadBalancerFormula(ElasticLoadBalancerFormula())
-                .computer(EbsEc2Instance(hardware.jira).withVolumeSize(300))
-                .databaseComputer(EbsEc2Instance(hardware.db).withVolumeSize(300))
-                .adminUser(scale.dataset.adminLogin)
-                .adminPwd(scale.dataset.adminPassword)
+                .computer(EbsEc2Instance(hardware.jira)).jiraVolume(Volume(300))
```

Comment:
[CI](https://server-gdn-bamboo.internal.atlassian.com/browse/QUICK-JHWRCI1-2) consistently fails with:
```
Error while executing tar -I lbzip2 -xf database.tar.bz2 -C 6cad46b9-4d14-4079-93f1-ae0594d9e727. Exit status code SshResult(exitStatus=2, output=, errorOutput=tar: database/ibdata1: Wrote only 9216 of 10240 bytes
```

Looks like the volumes are too small.
I just started a new run and I can see in CloudFormation:
![image](https://user-images.githubusercontent.com/1194704/61643260-c6b5e680-aca2-11e9-868c-01aa0d15194d.png)

It seems it uses 100 GB anyway. Investigating...

Reply:
So the 100 GB is the default and the `DataCenterFormula.Builder` has [a copying constructor, but does not copy over the`jiraVolume` and `databaseVolume` fields](https://github.com/atlassian/aws-infrastructure/blob/e248261c7c1e72b01b90ed9c99bdecf59a704a4d/src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt#L316-L330).

Raising an issue and fixing...